website/docs/d/is_vpn_server.html.markdown

```diff
@@ -0,0 +1,150 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_vpn_server"
+description: |-
+  Get information about VPNServer
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_vpn_server
+
+Provides a read-only data source for VPNServer. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_vpn_server" "is_vpn_server" {
+	identifier = "id"
+}
+```
+
+## Argument Reference
+Review the argument reference that you can specify for your data source.
+
+- `identifier` - (Optional, String) The ID of the VPN server.
+- `name` - (Optional, String) The name of the VPN server.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `identifier` - The unique identifier of the VPNServer.
```

Comment:
update the docs with the standard doc style.
* -> -

Reply:
done.